src/Serializer/DefaultEventSerializer.php

```diff
@@ -53,6 +53,11 @@ public function deserialize(SerializedEvent $data, array $options = []): object
         return $this->hydrator->hydrate($class, $payload);
     }
 
+    public function eventRegistry(): EventRegistry
```

Comment:
I'm not in favor of adding this method, just to have it easier in the tests. The `createFromPaths` is there to ease the creation and usage of this class by creating the object with reasonable default dependencies but this makes accessing the internal dependencies harder. Adding then methods to access these deps is not the solution imho, the solution should be to create this class manually then and not relying on `createFromPaths`.

The bare minimum should be marking this internal, as no user should rely on this.